src/Components/App/App.css

```diff
@@ -1,4 +1,14 @@
 * {
   margin: 0;
   padding: 0;
+}
+
+main {
+  background-color: #2f6a31;
```

Comment:
I have been putting the background .css in the index.css and covers the site better.